### PR TITLE
[EDU-3187] feat: add load balance a dns

### PR DESCRIPTION
**Pull request by hannah-sk**
### Changes

* Fixed title of the Activate DNSSEC guide.
* Added new page on how to perform a DNS load balance.

[EDU-3187]: https://aziontech.atlassian.net/browse/EDU-3187?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/load-balance-dns.mdx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+4. Click the **Add Record** button.
+5. In **Name**, provide the new record's name. Example: `load balance dns record`.
+6. In **Type**, select the type of record `A`.
+7. In **Value**, add the items for the DNS response to the registered record in IPv4 address format. Example: `192.111.0.1`.
+8. In **TTL (seconds)**, choose the time, in seconds, that a response can be cached on a resolver server. Maximum value: `2147483647`.
+9. In **Policy**, select **Weighted**.
+    - In **Weight**, specify the weight for the record, considering that you'll create other records for the same zone with different weights. Accepts values from `0` to `255`.
+    - In **Description** (optional), you can add a short text that differentiates the records you'll create for the load balance. Accepts up to *45 characters*.
+10. Click the **Save** button.
+11. Repeat steps 4 to 10 as many times as you want, but changing the **Weight** field.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/load-balance-dns.mdx:32]:
```suggestion
11. Repeat steps 4 to 10 as many times as you want, but changing the **Value** and **Weight** fields.
```

File: src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/load-balance-dns.mdx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    "id": 12345,
+    "description": "policy weight 50"
+  },
+  "schema_version": 3
+}
+```
+
+Wait a few minutes for the changes to propagate and your records will be created in the hosted zone you chose.
+
+5. Repeat the steps as many times as you want, but changing the **Weight** parameter.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/load-balance-dns.mdx:95]:
```suggestion
5. Repeat the steps as many times as you want, but changing the **answers_list** and **weight** parameters.
```

**Review by arturdj — CHANGES_REQUESTED** [review]

File: src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/load-balance-dns.mdx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+5. In **Name**, provide the new record's name. Example: `load balance dns record`.
+6. In **Type**, select the type of record `A`.
+7. In **Value**, add the items for the DNS response to the registered record in IPv4 address format. Example: `192.111.0.1`.
+8. In **TTL (seconds)**, choose the time, in seconds, that a response can be cached on a resolver server. Maximum value: `2147483647`.
+9. In **Policy**, select **Weighted**.
+    - In **Weight**, specify the weight for the record, considering that you'll create other records for the same zone with different weights. Accepts values from `0` to `255`.
+    - In **Description** (optional), you can add a short text that differentiates the records you'll create for the load balance. Accepts up to *45 characters*.
+10. Click the **Save** button.
+11. Repeat steps 4 to 10 as many times as you want, but changing the **Value** and **Weight** fields.
+    - Optionally, you can choose different values for the **Name** and **Description** fields.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/load-balance-dns.mdx:33]:
```suggestion
11. Repeat steps 4 to 10 to create the other balanced DNS records needed while observing Intelligent DNS limits, and always using the same **Name** and setting the desired addresses in **Value** and weights for each record in **Weight**.
```

File: src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/load-balance-dns.mdx [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+3. Run the following `POST` request, replacing `[TOKEN VALUE]` with your [personal token](/en/documentation/products/guides/personal-tokens/) and the `<hosted_zone_id>` value you copied:
+
+```bash
+curl --location 'https://api.azionapi.net/intelligent_dns/<hosted_zone_id>/records' \
+--header 'Content-Type: application/json' \
+--header 'Authorization: Token [TOKEN VALUE]' \
+--header 'Accept: application/json; version=3' \
+--data-raw '{
+    "record_type": "A",
+    "entry": "load balance dns record",
```
Comment (hannah-sk) [inline src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/load-balance-dns.mdx:59]:
```suggestion
    "entry": "lbexample",
```

File: src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/load-balance-dns.mdx [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+    "answers_list": [
+        "192.111.0.1",
+        "192.111.0.2"
+    ],
+    "ttl": 20,
+    "policy": "weighted",
+    "weight": 50
```
Comment (hannah-sk) [inline src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/load-balance-dns.mdx:66]:
```suggestion
    "weight": 50,
```

File: src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/load-balance-dns.mdx [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+    "description": "policy weight 50"
+  },
+  "schema_version": 3
+}
+```
+
+Wait a few minutes for the changes to propagate and your records will be created in the hosted zone you chose.
+
+5. Repeat the steps as many times as you want, but changing the **answers_list** and **weight** parameters.
+    - Optionally, you can choose different values for the **Name** and **Description** parameters.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/load-balance-dns.mdx:96]:
```suggestion
```

File: src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/load-balance-dns.mdx [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+    "id": 12345,
+    "description": "policy weight 50"
+  },
+  "schema_version": 3
+}
+```
+
+Wait a few minutes for the changes to propagate and your records will be created in the hosted zone you chose.
+
+5. Repeat the steps as many times as you want, but changing the **answers_list** and **weight** parameters.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/load-balance-dns.mdx:95]:
```suggestion
5. Repeat the steps to create the other balanced DNS records needed while observing Intelligent DNS limits, and always using the same **entry** and setting the desired addresses in the **answers_list** parameter and weights for each record in the **weight** parameter.
```

File: src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/load-balance-dns.mdx [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+
+---
+
+## Via RTM
+
+1. [Access RTM](/en/documentation/products/guides/how-to-access-rtm/) > **Intelligent DNS**.
+2. From the list, choose the zone in which you want to add records. Or, [create a zone](/en/documentation/products/guides/secure/intelligent-dns-configure-main-settings/#creating-a-new-zone).
+3. Select the **Records** tab.
+4. Click the **Add Record** button.
+5. In **Name**, provide the new record's name. Example: `load balance dns record`.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/load-balance-dns.mdx:24]:
```suggestion
5. In **Name**, provide the new record's name. Example: `lbexample`.
```

File: src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/load-balance-dns.mdx [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+```
+
+Wait a few minutes for the changes to propagate and your records will be created in the hosted zone you chose.
+
+5. Repeat the steps as many times as you want, but changing the **answers_list** and **weight** parameters.
+    - Optionally, you can choose different values for the **Name** and **Description** parameters.
+
+:::tip
+Check the [Azion API documentation](https://api.azion.com/) and the [OpenAPI specification](https://github.com/aziontech/azionapi-openapi/) to know more about what the Azion API can offer.
+:::
```
Comment (hannah-sk) [inline src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/load-balance-dns.mdx:100]:
```suggestion
:::

---

## Testing your balanced record

After configuring your balanced record, you can test and see if it's working via terminal commands:

1. Run `dig +short [your balanced hostname]` a couple of times in a row.
- You'll see the address list vary based on the record selected by the Intelligent DNS load balancer each time.
2. The response will be similar to:

```bash
dig +short lbexample @ns1.aziondns.net
192.111.0.1
```

```bash
dig +short lbexample @ns1.aziondns.net
192.111.0.2
```

```bash
dig +short lbexample @ns1.aziondns.net
192.111.0.1
```
```

**Review by arturdj — APPROVED** [review]

File: src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/load-balance-dns.mdx [hunk 10]
```diff
@@ -0,0 +1,116 @@
...
+description: Perform a DNS load balance by assigning different weights to records configured in Intelligent DNS.
+meta_tags: dns, intelligent dns, load balance, records
+namespace: docs_secure_load_balance_dns
+menu_namespace: secureMenu
+permalink: /documentation/products/guides/secure/load-balance-dns/
+---
+
+import ContributorList from '~/components/ContributorList.astro'
+
+You can use a [Intelligent DNS zone](/en/documentation/products/secure/intelligent-dns/) to create more than one record and perform a DNS load balance. Assigning different weights to each record for a same zone helps distribute incoming network traffic, ensuring optimal resource utilization and preventing overload on a single server.
```
Comment (gabriel-azion) [inline src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/load-balance-dns.mdx:12]:
```suggestion
You can use a [Intelligent DNS zone](/en/documentation/products/secure/intelligent-dns/) to create multiple records and perform a DNS load balance. Assigning different weights to each record for the same zone helps distribute incoming network traffic, ensuring optimal resource utilization and preventing overload on a single server.
```

File: src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/load-balance-dns.mdx [hunk 11]
```diff
@@ -0,0 +1,116 @@
...
+4. Click the **Add Record** button.
+5. In **Name**, provide the new record's name. Example: `lbexample`.
+6. In **Type**, select the type of record `A`.
+7. In **Value**, add the items for the DNS response to the registered record in IPv4 address format. Example: `192.111.0.1`.
+8. In **TTL (seconds)**, choose the time, in seconds, that a response can be cached on a resolver server. Maximum value: `2147483647`.
+9. In **Policy**, select **Weighted**.
+    - In **Weight**, specify the weight for the record, considering that you'll create other records for the same zone with different weights. Accepts values from `0` to `255`.
+    - In **Description** (optional), you can add a short text that differentiates the records you'll create for the load balance. Accepts up to *45 characters*.
+10. Click the **Save** button.
+11. Repeat steps 4 to 10 to create the other balanced DNS records needed while observing Intelligent DNS limits, and always using the same **Name** and setting the desired addresses in **Value** and weights for each record in **Weight**.
```
Comment (gabriel-azion) [inline src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/load-balance-dns.mdx:32]:
```suggestion
11. Repeat steps 4 to 10 to create the other balanced DNS records needed while observing Intelligent DNS limits, always using the same **Name** and setting the desired addresses in **Value** and weights for each record in **Weight**.
```

**Review by gabriel-azion — APPROVED** [review]
some suggestions. Feel free to accept or not... LGTM

File: src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/load-balance-dns.mdx (hunk 10 above)
Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/load-balance-dns.mdx:12]:
```suggestion
You can use an [Intelligent DNS zone](/en/documentation/products/secure/intelligent-dns/) to create more than one record and perform a DNS load balance. Assigning different weights to each record for a same zone helps distribute incoming network traffic, ensuring optimal resource utilization and preventing overload on a single server.
```

**Review by MarianaAguilera — APPROVED** [review]

File: src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/load-balance-dns.mdx [hunk 12]
```diff
@@ -0,0 +1,128 @@
...
+    "id": 61752,
+    "description": "policy weight 50"
+  },
+  "schema_version": 3
+}
+```
+
+Wait a few minutes for the changes to propagate and your records will be created in the hosted zone you chose.
+
+5. Repeat the steps to create the other balanced DNS records needed while observing Intelligent DNS limits, and always using the same **entry** and setting the desired addresses in the **answers_list** parameter and weights for each record in the **weight** parameter.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/load-balance-dns.mdx:94]:
```suggestion
5. Repeat the steps to create the other balanced DNS records needed while observing Intelligent DNS limits, always using the same `entry` and setting the desired addresses in the `answers_list` parameter and weights for each record in the `weight` parameter.
```

**Review by bru-andrade — APPROVED** [review]

File: src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/load-balance-dns.mdx [hunk 13]
```diff
@@ -0,0 +1,128 @@
...
+
+1. Run the following `GET` request in your terminal, replacing `[TOKEN VALUE]` with your [personal token](/en/documentation/products/guides/personal-tokens/) to retrieve your `<hosted_zone_id>`:
+
+```bash
+curl --location 'https://api.azionapi.net/intelligent_dns' \
+--header 'Authorization: Token [TOKEN VALUE]' \
+--header 'Accept: application/json; version=3'
+```
+
+2. You'll receive a response with all your existing edge firewalls. Copy the value of the `<id>` that you want to use.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/secure-journey/intelligent-dns-advanced-configurations/load-balance-dns.mdx:48]:
```suggestion
2. You'll receive a response with all your existing zones. Copy the value of the `<id>` that you want to use.
```

**Pull request merged**